docs/operator_parse.md

```diff
@@ -0,0 +1,268 @@
+    property: 'events'
+}
+
+// output
+{
+    events: [ 'event1', 'event2', 'event13', 'purchase' ]
+}
+```
+### Key/Value Parsing
+Demonstrating how key/value parsing works with simply properties getting `null` values.   
+```javascript
+data = {
+    events: 'event1,event2=foo',
+}
+operator = {
+    name: 'parse',
+    propertyDelimiters: [','],
+    keyValueDelimiter: '=',
+    property: 'events',
+}
+
+// output
+{
+    event1: null,
+    event2: 'foo'
+}
+```
+### Invalid Property Names
+If you use key/value parsing, but have propert(ies) that are not [valid](https://help.fullstory.com/hc/en-us/articles/360020623234#property-name-requirements) they will be placed into an array of the original property name. 
+```javascript
+data = {
+    events: 'event1,12Foo,Hello_1234,_data=some,_foo,event2=foo,3piece=bar',
+}
+operator = {
+    name: 'parse',
+    propertyDelimiters: [','],
+    keyValueDelimiter: '=',
+    property: 'events'
+};
+
+// output
+{
+    event1: null,
+    Hello_1234: null,
+    event2: 'foo'
+    events: [ '12Foo', '_data=some', '_foo', '3piece=bar' ]
+}
+```
+
+### Advanced Scenario
+This example will show an advanced scenario with multiple `propertyDelimiters`, invalid property names, etc. 
+```javascript
+ data = {
+    products: 'Example;1;3.50;event1=4.99|event2=5.99;eVar1=Example value 1|eVar2=Example value 2',
+}
+
+operator = {
+    name: 'parse',
+    propertyDelimiters: [';', '|'],
+    keyValueDelimiter: '=',
+    property: 'products',
+}
+
+// output
+{
+    Example: null,
+    event1: 4.99,
+    event2: 5.99,
+    eVar1: 'Example value 1',
+    eVar2: 'Example value 2',
+    events: [ '1', '3.50' ]
+}
+```
+## Full DLO Examples
+This section will show some real-world examples of parsing inside full DLC rules.  
+
+### Rule
+
+```json
+{
+    "id": "simple", 
+    "source":"s[(events)]", 
+    "operators":
+    [{
+        "name": "parse",
+        "parseDelimiters": "[ ',' ]",
+        "keyValueDelimiter": "=",
+        "property": "events"
+      },
+      {
+            "name": "query",
+            "select": "$[(event13)]"
+      },
+      {
+            "name": "insert",
+            "value": "Event Initiated"
+      }],
+        "destination":  "FS.event"
+}    
+```
+
+### Input
+
+```javascript
+s {
+  events: 'event1,event2=foo,event13,purchase',
+  products: 'Example product;1;event1=4.99|event2=5.99;eVar1',
+}
+```
+
+### Output
+
+```json
+[
+ "Event Initiated",
+ {}
+]
+```
+
+
+### Rule
+
+```json
+{
+  "id": "complex",
+  "source": "s[(events)]",
+  "operators": [{
+      "name": "parse",
+    "parseDelimiters": "[ ‘,’ ]",
+    "keyValueDelimiter": "=",
+    "property": "events"
+  },
+  {
+      "name": "query",
+      "select": "$[(event2)]"
+  },
+  {
+      "name": "rename",
+      "properties": {
+        "event2": "amount"
+      }
+  }, 
+  {
+      "name": "insert",
+      "value": "Purchase"
+   }
+
+  ],
+  "destination": "FS.event"
+}
+```
+
+### Input
+
+```javascript
+ s {
```

Comment:
This and the example below have an extra space at the beginning that isn't present in the first one.

Reply:
Removed extra space